<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgPaletteDock</class>
 <widget class="QWidget" name="WdgPaletteDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>647</width>
    <height>422</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="KisPaletteView" name="paletteView">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>16</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="KisPaletteComboBox" name="cmbNameList">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="7">
      <widget class="QToolButton" name="bnRemove">
       <property name="toolTip">
        <string>Delete color</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-remove">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <spacer name="hBottomSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="5">
      <widget class="QToolButton" name="bnRename">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="edit-rename">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="KisPopupButton" name="bnColorSets">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="4">
      <widget class="QToolButton" name="bnAdd">
       <property name="toolTip">
        <string>Add foreground color</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="list-add">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>22</width>
         <height>22</height>
        </size>
       </property>
       <property name="autoRaise">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="6">
      <widget class="QToolButton" name="bnEditPalette">
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset theme="document-layer">
         <normaloff>.</normaloff>.</iconset>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="KSqueezedTextLabel" name="lblPaletteName">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisPopupButton</class>
   <extends>QPushButton</extends>
   <header>KisPopupButton.h</header>
  </customwidget>
  <customwidget>
   <class>KisPaletteView</class>
   <extends>QTableView</extends>
   <header location="global">kis_palette_view.h</header>
  </customwidget>
  <customwidget>
   <class>KisPaletteComboBox</class>
   <extends>QComboBox</extends>
   <header>KisPaletteComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>KSqueezedTextLabel</class>
   <extends>QLabel</extends>
   <header location="global">ksqueezedtextlabel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>bnRemove</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
